build.sh

```diff
@@ -5,7 +5,7 @@ set -e
 
 cd "$(dirname "$0")"
 
-PYTHON_VER=3.9.6
+PYTHON_VER=3.9.13
```

Comment:
Separate commits for unrelated changes. Python version bump should not be bundled with a shell script change, if at all.